include/net/download_client.rst

```diff
@@ -132,6 +134,12 @@ The following snippet illustrates how to provision a TLS certificate, associate
 	}
 
 
+Limitations
+***********
+
+The library requires the host server to provide a Content-Range field in the HTTP GET header.
+If this header field is missing, the library returns an error.
```

Comment:
What error do you get?

Reply:
The library will log an error
```
[00:00:09.354,339] <err> download_client: Server did not send "Content-Range" in response
```

Reply:
Recommended path for debugging when getting such error would be to turn the log level of the `download_client` to debug and select `DOWNLOAD_CLIENT_LOG_HEADERS`

```
CONFIG_DOWNLOAD_CLIENT_LOG_LEVEL_DBG=y
CONFIG_DOWNLOAD_CLIENT_LOG_HEADERS=y
```

In `prj.conf`